doc/source/examples/multiphysics/bubble-detachment-shear-flow/bubble-detachment-shear-flow.rst

```diff
@@ -0,0 +1,417 @@
+==================================
+Bubble detachment in shear flow
+==================================
+
+This example simulates the detachment of a growing bubble of air in a liquid shear flow.
+
+
+----------------------------------
+Features
+----------------------------------
+
+- Solver: ``lethe-fluid`` (Q1-Q1)
+- Two phase flow handled by the Cahn-Hilliard-Navier-Stokes (CHNS) approach
+- Dimensionality of the length
```

Comment:
```suggestion
- Dimensionality of the length
```
I understand what you mean here and I don't know how to word that better, but I still think the wording here could be improved